operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md

```diff
@@ -4,7 +4,7 @@ Non-compute node (NCN) personalization applies post-boot configuration to the
 HPE Cray EX management nodes. Several HPE Cray EX product environments outside
 of CSM require NCN personalization to function. Consult the manual for each
 product to configure them on NCNs by referring to the
-[1.5 HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-gsg)
+[1.5 HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-S-8000)
```

Comment:
Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
[HPE Cray EX System Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-S-8000)
```